src/fieldlist_OMIP_CESM2.jsonc

```diff
@@ -0,0 +1,28 @@
+// All text to the right of an unquoted "//" is a comment and ignored, as well
```

Comment:
You don't need to make edits to the fieldlist files:
- Adding variable names is our responsibility (issue #134).
- Coordinate names can be set to whatever you like via the new settings format.